README.md

```diff
@@ -60,23 +60,25 @@ env:
-| `calc`                | Comma-separated list of metrics to be used. GATE metric is supported for TiCS Viewer versions higher than 2022.2.x. If not specified, `GATE` will be used by default.                                                | false    |
-| `clientData`          | A custom client-data token for the purpose of the Client Viewer functionality. This provides a static URL that is updated with every analysis.                                                                       | false    |
+| Input                  | Description                                                                                                                                                                                                          | Required |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `githubToken`          | Provided by Github automatically in an action (see [Authenticating with the GITHUB_TOKEN](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token))     | true     |
+| `projectName`          | Name of the TiCS project present in the TiCS Viewer.                                                                                                                                                                 | true     |
+| `ticsConfiguration`    | A URL pointing to the "cfg" API endpoint of the TiCS Viewer. It contains the name of the TiCS Analyzer Configuration or "-" in case of the default configuration.                                                    | true     |
+| `branchName`           | Name of the branch in TiCS.                                                                                                                                                                                          | false    |
+| `branchDir`            | Location of the files to analyze.                                                                                                                                                                                    | false    |
+| `calc`                 | Comma-separated list of metrics to be used. GATE metric is supported for TiCS Viewer versions higher than 2022.2.x. If not specified, `GATE` will be used by default.                                                | false    |
+| `recalc`               | Comma-separated list of metrics to be recalculated. GATE metric is supported for TiCS Viewer versions higher than 2022.2.x.                                                                                          | false    |

```

Comment:
Perhaps we should specify, since both calc and recalc are optional, that if calc and recalc are not specified, the default is taken, e.g. CODINGSTANDARD (CS).